### ISSUE TITLE
Different name for keypass db files

### ISSUE THREAD

**Issue by garpinc** [perfectapi/CKP#96]
My keypass files on onedrive are located in a subdirectory and are called [filename].kdbx.doc. This works for my other clients but for whatever reason CKP does not find these files and it provides me with no option to select them either.

**Comment by DukeyToo** [comment]
Yes, CKP will only show files with the expected extension.  This is intended behavior.  

It would not be a difficult change to support the pattern of filename you are using.  I would accept a PR that did it.

**Comment by tmajibon** [comment]
Given that other people would take strategies of hiding the extension, perhaps an option to show all files or provide a search function? (just type in a string, click search, and use that string as the pattern recognition)

**Comment by ciaran-byrne** [comment]
I'd suggest keeping just the .kdb* extension; if I'm presented with all the *.doc files (or any other extension) that I've stored my list could run to the thousands, and I wouldn't fancy having to scroll through them all
